sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -148,18 +140,18 @@ namespace sycl::ext::oneapi::experimental::matrix {
   template <typename Group, typename T, size_t NumRows, size_t NumCols,
             access::address_space Space>
   void joint_matrix_load(Group sg,
-    joint_matrix<T, use::accumulator, NumRows, NumCols, layout::dynamic, Group> &res,
+    joint_matrix<Group, T, use::accumulator, NumRows, NumCols, layout::dynamic> &res,
     multi_ptr<T, Space, IsDecorated> src, size_t stride, layout memL);
     
   template <typename Group, typename T, size_t NumRows, size_t NumCols,
           use Use, layout Layout, access::address_space Space>
   void joint_matrix_load(Group sg,
-    joint_matrix<T, Use, NumRows, NumCols, Layout, Group> &res,
+    joint_matrix<Group, T, Use, NumRows, NumCols, Layout> &res,
     multi_ptr<T, Space, IsDecorated> src, size_t stride);
 }
 ```
 
-`joint_matrix_load` loads data from memory to the 2d tiles/registers of Intel AMX/DPAS.
+`joint_matrix_load` loads data from memory to the 2d tiles/registers of the tensor hardware.
 We define two overloads of the load function depending on whether the memory layout was declared as part of the `joint_matrix` type or not. 
 The first overload that takes memory layout as an argument is only available for a `joint_matrix` type that was declared with `layout::dynamic`.
```

Comment:
```suggestion
The first overload that takes memory layout as an argument is only available for a `joint_matrix` type that used the default value `layout::dynamic`.
```

I think all references to `layout::dynamic` should emphasize that it is only intended to be used as a default value, and users should never have to type `layout::dynamic`